tasks/README.md

```diff
@@ -0,0 +1,96 @@
+# Tasks
+
+## Task object
+
+```
+{
+  "type":"Dev | Group",
+  "links": [
+    <link1>,
+    <link2>
+  ],
+  "endsOn":"<end date>",
```

Comment:
What is the format of the date?

Reply:
Disucssed the format with @ankurnarkhede on this , the format will be unix timestamp.